platform/macos/display_server_macos.mm

```diff
@@ -3609,7 +3609,11 @@
 			gl_manager_angle = nullptr;
 			bool fallback = GLOBAL_GET("rendering/gl_compatibility/fallback_to_native");
 			if (fallback) {
+#ifdef EGL_STATIC
 				WARN_PRINT("Your video card drivers seem not to support the required Metal version, switching to native OpenGL.");
+#else
+				WARN_PRINT("Your video card drivers seem not to support the required Metal version or ANGLE dynamic libraries (libEGL.dylib and libGLESv2.dylib) are missing, switching to native OpenGL.");
+#endif
```

Comment:
These messages are a bit confusing as they talk about Metal for an OpenGL driver. I assume it means ANGLE failed to initialize due to lacking proper Metal support. I think it should be rephrased to say that (the first warning doesn't mention ANGLE at all).

Reply:
I guess it can be simply `... not to support GLES3 / ANGLE`, since the same message is shown regardless of the reason of error (ANGLE GL manager will print its own error before it).